exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/StatusResources.java

```diff
@@ -50,75 +53,124 @@
 public class StatusResources {
   static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(StatusResources.class);
 
+  public static final String PATH_STATUS_JSON = "/status.json";
+  public static final String PATH_STATUS = "/status";
+  public static final String PATH_OPTIONS_JSON = "/options.json";
+  public static final String PATH_INTERNAL_OPTIONS_JSON = "/internal_options.json";
+  public static final String PATH_OPTIONS = "/options";
+  public static final String PATH_INTERNAL_OPTIONS = "/internal_options";
```

Comment:
Nice improvement to externalize constants. Perhaps the constants can show the relationships between paths. For example:

```
public static final String REST_API_SUFFIX = ".json";
public static final String PATH_STATUS_UI = "/status";
public static final String PATH_STATUS_API = PATH_STATUS_UI + REST_API_SUFFIX;
```